.github/PROCESS.md

```diff
@@ -85,6 +86,57 @@ We have two long-living branches:
 - `master`: completed features, bug fixes, refactors, chores
 - `stable`: the latest release
 
+The overall flow:
+
+1. A `release` branch is created from `master`
+1. Feature, refactor, and bug fix branches are created from `master`
+1. When a feature, refactor, or fix is complete it is merged into `master`
+1. When the release branch is done it is merged into `master` and `stable`
+1. If an issue in `stable` is detected a `hotfix` branch is created from `stable`
+1. Once the `hotfix` is complete it is merged to both `master` and `stable`
+1. All branches should follow the syntax of `{type}-{details}` where `{type}` is the type of branch (`hotfix`, `release`, or one of the [commit types](https://github.com/ionic-team/ionic/blob/master/.github/CONTRIBUTING.md#commit-message-format)) and `{details}` is a few hyphen separated words explaining the branch
+
+
+### Stable and Master Branches
+
+#### Stable Branch
+
+Branches created from `stable`:
+
+- A `hotfix` branch (e.g. `hotfix-missing-export`) should be the *only* branch that is created from stable and it gets merged back into both `master` and `stable` when complete
+
+#### Master Branch
+
+Branches created from `master`:
+
+The following branches should be created from `master` and merged back to `master` via a pull request:
+
+1. A feature branch (e.g. `feat-desktop-support`): an addition to the API that is not a bug fix or regression fix
+1. A bug fix branch (e.g. `fix-tab-color`): a bug fix that is not fixing a regression or issue with a published release
+1. All other types listed in the [commit message types](https://github.com/ionic-team/ionic/blob/master/.github/CONTRIBUTING.md#commit-message-format): `docs`, `style`, `refactor`, `perf`, `test`, `chore`
+
+The following branch should be created from `master` and merged back to *both* `master` and `stable`:
+
+1. A `release` branch (e.g. `release-4.1.x`): contains all fixes and (optionally) features that are tested and should go into the release
+
+
+### Feature Branches
+
+Each new feature should reside in its own branch, based on the `master` branch. When a feature is complete, it should go into a pull request that gets merged back into `master`. Features should never interact directly with `stable`.
+
+
+### Release Branches
+
+Once `master` has acquired enough features for a release (or a predetermined release date is approaching), you fork a release branch off of `master`. Creating this branch starts the next release cycle, so no new features can be added after this point - only bug fixes, documentation generation, and other release-oriented tasks should go in this branch.
+
+Once the release is ready to ship, it will get merged into `stable` and `master`, then the release branch will be deleted. It’s important to merge back into `master` because critical updates may have been added to the release branch and they need to be accessible to new features. This should be done in a pull request after review.
```

Comment:
At what point do we publish to npm? is that covered somewhere else?

Reply:
nvm I see it at the bottom. Maybe make a reference to this somewhere in here? (not required, just thought it might be nice to have)